.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: 💬 Questions / Help
+labels: ':speech_balloon: Question'
+about: If you have questions, please check StackOverflow
+---
+
+## 💬 Questions and Help
+
+### Please note that this issue tracker is not a help forum and this issue will be closed.
+
+For questions or help please see:
+
+- [The jest-preset-angular online documentation](https://github.com/thymikee/jest-preset-angular/blob/master/README.md)
+- The [jest-preset-angular](https://stackoverflow.com/questions/tagged/jest-preset-angular) tag on [StackOverflow](https://stackoverflow.com/questions/ask)
```

Comment:
For now we were pretty open for help questions, are we going to restrict more in the future?

Reply:
The template for question is more like just for structuring issues. 

Maybe the instruction text can be changed in such a way that it is not mandatory to check on StackOverflow ?

Reply:
Actually I am ok with moving issues towards SO, but I would have to subscribe to our tag then :sweat_smile:

Reply:
I think we should use both channels, GitHub and SO. Maybe the text can be something like:

“For questions or help, you can also check on StackOverflow..” ?

Reply:
I modified this text to the one I mentioned above.